### PR TITLE
gpsd: update to 3.21

### PR DESCRIPTION
**Pull request by neheb**
Signed-off-by: Rosen Penev <rosenp@gmail.com>

Maintainer: @psidhu 
Compile tested: ath79

ping @jefferyto

### CODE REVIEW

**Comment by aparcar** [comment]
Please add a runtime test

**Comment by neheb** [comment]
Test failed. How do you redirect from stderr to stdout?

File: utils/gpsd/test.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+cgps -V 2>&1 | grep "$2"
```
Comment (aparcar) [inline utils/gpsd/test.sh:3]:
```suggestion
gpsd -V 2>&1 | grep "$2"
```

**Review by aparcar — CHANGES_REQUESTED** [review]

File: utils/gpsd/test.sh (hunk 1 above)
Comment (neheb) [reply utils/gpsd/test.sh:3]: gpsd does not have a version option.

Comment (aparcar) [reply utils/gpsd/test.sh:3]:
```shell
root@dba40b1634ab:/# gpsd -V
gpsd: 3.20 (revision 3.20)
```

Comment (aparcar) [reply utils/gpsd/test.sh:3]: So this package results in both a `gpsd` and a `gpsd-clients` package. To test it a script like the following should work:

```shell
case "$1" in
	"gpsd")
		gpsd -V 2>&1 | grep "$2"
		;;
        "gpsd-clients")
                cgps -V 2>&1 | grep "$2"
                ;;
esac
```

Comment (neheb) [reply utils/gpsd/test.sh:3]: Hmm maybe it was changed. gpsd -V locally returns nothing (version 3.19).

File: utils/gpsd/test.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+case "$1" in
+	"gpsd")
+		gpsd -V 2>&1 | grep "$2"
+		;;
+	"gpsd-clients"
```
Comment (aparcar) [inline utils/gpsd/test.sh:7]:
```suggestion
	"gpsd-clients")
```

SORRY

**Review by aparcar — CHANGES_REQUESTED** [review]

**Review by aparcar — APPROVED** [review]

**Comment by neheb** [comment]
Merging. The buildbots seem happy.

**Pull request merged**